tools/cli-bootstrap/cli.bootstrap.py

```diff
@@ -0,0 +1,562 @@
+#!/usr/bin/env python 
+
+import os
+import json
+import platform
+import argparse
+import sys
+import traceback
+
+# for readability
+from subprocess import call
+from subprocess import check_output
+from subprocess import check_call
+from subprocess import CalledProcessError
+
+from os import path
+from os import makedirs
+
+from string import find
+from urllib import urlretrieve
+
+
+# ROVER BASE #
+class RoverMods:
+    HEADER = '\033[95m'
```

Comment:
I just tested and this works on an alpine container for me. core-setup and cli have their own variant of console color, and it doesn't work in most docker containers. There's also some shell scrpts around that don't work unless you install stuff that they use for console coloring.

But this works, so I'm cool with it.